docs/how-to/default-profile.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Default profile
+legacyUrl: https://docs.ipfs.io/guides/examples/default-profile/
+description: Hands-on guide for rolling back to the default profile
+---
+
+# Configure a default profile and switching between profiles
+
+---
+### What is a profile
```

Comment:
```suggestion
```

Since this is an _introduction_ paragraph that's explaining what profiles do, it doesn't need to come under it's own sub-title.